storage/database/db_manager.go

```diff
@@ -140,34 +140,33 @@ type DBManager interface {
 	ReadCode(hash common.Hash) []byte
 	ReadCodeWithPrefix(hash common.Hash) []byte
 	WriteCode(hash common.Hash, code []byte)
+	PutCodeToBatch(batch Batch, hash common.Hash, code []byte)
```

Comment:
PutCodeToBatch?
WriteCodeToBatch?

Reply:
Yes. I did sway between the two. But I decided to go with PutCodeToBatch.

Take "Body" schema as an example,
- klaytn 1.10.2 DBManager has both `WriteBody(...)` and `PutBodyToBatch(batch, ...)`
- [geth 1.11.5 eth/rawdb](https://github.com/ethereum/go-ethereum/blob/v1.11.5/core/rawdb/accessors_chain.go) has only `WriteBody(KVwriter, ...)` because they somehow treat LevelDB and Batcher equally.

To follow the current convention, PutCodeToBatch is a better fit.

Reply:
This `Put` method is a bit awkward to me. There is no other put method for batch in dbmanager except for this. On the other hand, I understood your point as well. Also, shouldn't we change `batch.Write` as well to something like `WriteBatches` method?

Reply:
There were a few `Put..ToBatch` methods before this PR: PutBodyToBatch, PutReceiptsToBatch, etc. I'm not sure about their origin because they don't exist in geth. Anyway, if we're to rename `Put..ToBatch` to something else, it should be another PR.

For replacing `batch.Write` to `WriteBatches(batch)`, I attempted but reverted because there are so many. Again, it should be another PR on its own.